src/sidebar.jsx

```diff
@@ -276,7 +275,7 @@ const DropdownWithKebab = ({
             id: "edit-permissions",
             onClick: () => {
                 editPermissions(Dialogs, {
-                    selected: selected[0] || currentDirectory,
+                    selected: selected[0] || { name: directory_name, type: "dir" },
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-256-20240212-150910-02953828-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)